database/txclient/tx_config.go

```diff
@@ -24,6 +27,7 @@ func NewTxConfig(gasLimit uint64, receiptTimeout time.Duration, receiptInterval
 func NewTransferTxConfig() TxConfig {
```

Comment:
[MINOR] It would reduce duplicity if you would have written:
```
return NewTxConfig(database.MinGasPrice, 21000, TxReceiptTimeout...)
```

Reply:
nice idea, done